test/Templates.Test/Helpers/SeleniumServerFixture.cs

```diff
@@ -0,0 +1,103 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using System;
+using System.Collections.Generic;
+using System.Diagnostics;
+using System.IO;
+using System.Management;
+using System.Runtime.InteropServices;
+
+namespace Templates.Test.Helpers
+{
+    public class SeleniumServerFixture : IDisposable
+    {
+        private string _workingDirectory;
+        private Process _serverProcess;
+        private static readonly object _serverLock = new object();
+
+        public SeleniumServerFixture()
+        {
+            _workingDirectory = Directory.GetCurrentDirectory();
+            StartSeleniumStandaloneServer();
+        }
+
+        public void Dispose()
+        {
+            // Find the java process running selenium-standalone
+            foreach (var childProcess in GetChildProcesses(_serverProcess.Id))
+            {
+                childProcess.Kill();
+                childProcess.Dispose();
+            }
+
+            _serverProcess.Kill();
+            _serverProcess.Dispose();
+        }
+
+        private static List<Process> GetChildProcesses(int parentProcessId)
```

Comment:
@SteveSandersonMS Thoughts on this? This gets all the children processes of the `_serverProcess`

Reply:
This is not cross platform. Doesn't work on Linux or Mac